libs/sdk-core/src/breez_services.rs

```diff
@@ -416,8 +411,7 @@ impl BreezServices {
     ///
     /// # Arguments
     ///
-    /// * `description` - The bolt11 payment request description
-    /// * `amount_sats` - The amount to receive in satoshis
+    /// * `req_data` - Request parameters for receiving a payment
     pub async fn receive_payment(
         &self,
         req_data: ReceivePaymentRequest,
```

Comment:
This can become `req` just like in `lnurl_withdraw(`

Reply:
done